src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/lynx/insights/quill-services/quill-editor-ready.service.spec.ts

```
@@ -0,0 +1,89 @@
+    }
+  }
+
+  // Compatible with rxjs 'fromEvent' (EventTarget-like)
+  addEventListener(eventName: string, callback: Function): void {
+    if (!this.eventHandlers[eventName]) {
+      this.eventHandlers[eventName] = [];
+    }
+    this.eventHandlers[eventName].push(callback);
+  }
+
+  removeEventListener(eventName: string, callback: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)